pkg/controller/container-runtime-config/container_runtime_config_controller.go

```diff
@@ -55,6 +55,11 @@ const (
 var (
 	// controllerKind contains the schema.GroupVersionKind for this controller type.
 	controllerKind = mcfgv1.SchemeGroupVersion.WithKind("ContainerRuntimeConfig")
+
+	defaultPools = map[string]bool{
+		"master": true,
+		"worker": true,
```

Comment:
Hm.  I'm OK with this for now but...feels like we should have a `machineconfiguration.openshift.io/mco-built-in: ""` annotation or something on the pools.

Reply:
that would be definitely better indeed